cores/arduino/stm32/usb/cdc/usbd_cdc_if.c

```diff
@@ -37,6 +37,7 @@
 USBD_HandleTypeDef hUSBD_Device_CDC;
 
 static bool CDC_initialized = false;
+static bool hostPortOpen = false;

```

Comment:
I wonder if it is really need to define a new variable.

Reply:
as written, no it is the same as the value lineState
I was reading this thread:
https://stackoverflow.com/questions/5338875/detecting-open-pc-com-port-from-usb-virtual-com-port-device

and there it was suggested to trigger the hostPortOpen on SET_LINE_CODING, but it was not clear what would reset it, so I left it out.

Reply:
As documented on many site:
> On many operating systems, including Windows, the DTR line is held low while the serial port is unused and not being controlled by any applications. 

What I wonder if the `bool ` operator should be updated to have the `while(!Serial)` functional.

Reply:
I was thinking that the bool() was intended to return isConnected().

Reply:
Right forgot isConnected ;)